actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 require "action_view/helpers/tag_helper"
-require "active_support/core_ext/string/access"
-require "i18n/exceptions"
+require "active_support/core_ext/object/blank"
```

Comment:
This isn't necessary, `blank?` is always available internally (except within Active Support itself):

https://github.com/rails/rails/blob/1b2879404fd763b41fea234d24b008247d6d52c8/actionview/lib/action_view.rb#L27

https://github.com/rails/rails/blob/1b2879404fd763b41fea234d24b008247d6d52c8/activesupport/lib/active_support/rails.rb#L5-L7

https://github.com/rails/rails/blob/1b2879404fd763b41fea234d24b008247d6d52c8/activesupport/lib/active_support/rails.rb#L13-L14